code/control/DNRoot.php

```diff
@@ -2339,4 +2346,99 @@ public function AmbientMenu() {
 		return $list;
 	}
 
+	/**
+	 * Create project action.
+	 *
+	 * @return SS_HTTPResponse
+	 */
+	public function createproject(SS_HTTPRequest $request) {
+		if($this->canCreateProjects()) {
+			return $this->render(['CurrentTitle' => 'Create Stack']);
+		}
+		return $this->httpError(403);
+	}
+
+	/**
+	 * Checks whether the user can create a project.
+	 *
+	 * @return bool
+	 */
+	protected function canCreateProjects() {
+		$project = Injector::inst()->get('DNProject');
+		return $project->allowed(self::ALLOW_CREATE_PROJECT);
```

Comment:
"allowed" class of permission checkers are bound to a project. Can you rename this permission to not use this keyword and use normal Permission::check?